example-config/etesync.ini

```diff
@@ -0,0 +1,18 @@
+# uwsgi configuration file
+# typical location of this file would be /etc/uwsgi/sites/etesync.ini
+
+[uwsgi]
+folder = /path/to/etesync
+
+socket = /run/uwsgi/etesync.sock
+chown-socket = EtesyncUser:www-data
+chmod-socket = 660
+vacuum = true
+
+
+uid = etes
```

Comment:
uid is for user id, so here you're using etesync, but above you're using EtesyncUser. Inconsistent.